### DISCUSSION TITLE
SAML 2.0 Support for SSO

### DISCUSSION THREAD

**Discussion by CovantaDataManagement** [Budibase/budibase#2036] [Ideas]
Has any thought been given to creating SAML 2.0 authentication support for SSO to our Active directory?

**Comment by Rory-Powell** [comment]
Hi @CovantaDataManagement. SAML support isn't directly on our roadmap, however we are almost finished the OpenID Connect integration - an authentication protocol that is supported by active directory. Would this work for you?

Here is a preview of the doc changes going out with this feature
https://github.com/Budibase/docs/pull/6/files?short_path=331159a#diff-331159abc9e3a8b30e7c68a5048225a3c6254325ef519c163beb70bbf8e4c04d

**Comment by oswaldB** [comment]
Hello 

+1 for the SAML connector.
